packages/espresso/android/src/main/java/androidx/test/espresso/flutter/internal/protocol/impl/DartVmServiceUtil.java

```diff
@@ -71,6 +71,7 @@ public static String getDartIsolateId(View flutterView) {
   }
 
   /** Gets the Dart executor for the given {@code flutterView}. */
+  @SuppressWarnings("deprecation")
```

Comment:
ditto here

Reply:
PR Updated with requested changes . Thank you